R/load_cell_graphs.R

```diff
@@ -312,7 +312,16 @@ LoadCellGraphs.MPXAssay <- function (
       cg <- cg_list_full[[nm]]
```

Comment:
I am wondering whether `load_precomputed_layouts` should be called `load_layouts` instead? It seems like the "precomputed" is making an unnecessary assumption about how the layouts ended up in the file. Just a side point.

Reply:
Yeah that's a fair point. But could it be that `load_layouts` leads users to think that they are computed in R?

Reply:
I will update the argument name to `load_layouts`!

Reply:
Now the script name has changed from `load_precomputed_layouts.R` to `load_layouts.R` and the argument passed to `LoadCellGraphs` is now called `load_layouts`.